app/src/main/java/net/gsantner/opoc/frontend/filebrowser/GsFileBrowserListAdapter.java

```diff
@@ -130,9 +133,39 @@ public GsFileBrowserListAdapter(GsFileBrowserOptions.Options options, Context co
             _dopt.folderColor = cu.getResId(context, GsContextUtils.ResType.COLOR, "folder");
         }
 
+        _virtualMapping = Collections.unmodifiableMap(getVirtualFolders());
+        _reverseVirtualMapping = Collections.unmodifiableMap(GsCollectionUtils.reverse(_virtualMapping));
         loadFolder(_dopt.startFolder != null ? _dopt.startFolder : _dopt.rootFolder, null);
     }
 
+    public Map<File, File> getVirtualFolders() {
```

Comment:
Populate virtual folders at start

Reply:
can't find the issue, but I recall there being topics about virtual lists not outdated